source/docs/romi-robot/imaging-romi.rst

```diff
@@ -3,44 +3,70 @@ Imaging your Romi
 
 .. warning:: We are working hard on getting the Romi documentation complete. Please check back later.
 
-The Romi has 2 boards that both need to be imaged in order for the robot to work – the Raspberry Pi and the Romi 32U4 Control Board. Once the Raspberry Pi is imaged, we can use the web dashboard included on the device to image the 32U4 Control Board.
+The Romi has 2 microprocessor boards:
+
+#. A **Raspberry Pi** that handles high level communication with the robot program running on the desktop and
+#. A **Romi 32U4 Control Board** that handles low level motor and sensor operation.
+
+Both boards need to have firmware installed so that the robot operate properly.
 
 Raspberry Pi
 ------------
 
 Download
 ^^^^^^^^
 
-The Raspberry Pi Romi image is built on top of WPILibPi.
+The Raspberry Pi firmware is based on the WPILibPi (formerly FRCVision) and must be downloaded and written to the Raspberry Pi micro SSD card. Information about the Raspberry Pi image can be found here. Click on ``Assets`` at the bottom of the description to see the available image files:
```

Comment:
```suggestion
The Raspberry Pi firmware is based on the WPILibPi (formerly FRCVision) and must be downloaded and written to the Raspberry Pi micro SD card. Information about the Raspberry Pi image can be found here. Click on ``Assets`` at the bottom of the description to see the available image files:
```

Additionally, should there be a link on ``here``

Reply:
@bradamiller What do you want `here` to link to?

Reply:
There's a lot of information on the release page, so you cold just use that